bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml

```diff
@@ -1070,7 +1070,22 @@ spec:
                 description: Format specifies the output format of the bootstrap data
                 enum:
                 - cloud-config
+                - ignition
                 type: string
+              ignitionConfig:
+                description: "Ignition contains Ignition specific configuration. \n If set, bootstrap data will be generated in Ignition format instead of cloud-init."
```

Comment:
```suggestion
                description: "Ignition contains Ignition configuration. \n If set, bootstrap data will be generated in Ignition format instead of cloud-init."
```

Reply:
Maybe `Ignition contains fields for customizing generated bootstrap data in Ignition format` ?

Reply:
@binchenX ?